docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+
+The addresses with the listed node IDs will not be sent to other peers, when the peer exchange reactor
+([`p2p.pex`](#p2ppex)) is enabled. This allows a more granular setting instead of completely disabling the peer exchange
+reactor.
+
+For example, sentry nodes in the
+[Sentry Node Architecture](https://forum.cosmos.network/t/sentry-node-architecture-overview/454) on the Cosmos Hub can
+use this setting to make sure they do not gossip the node ID of the validator node, while they can still accept node
+addresses from the Internet.
+
+### p2p.allow_duplicate_ip
+Toggle to disable guard against peers connecting from the same ip.
+```toml
+allow_duplicate_ip = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+When this setting is set to `true`, multiple connections are allowed from the same IP address (for example on different
+ports).
+
+### p2p.handshake_timeout
+Timeout duration for protocol handshake (or secret connection negotiation).
+```toml
+handshake_timeout = "20s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+This high-level timeout value is applied when the TCP connection has been made and the peers are negotiating an upgrade
+to secret connection.
+
+The value `"0s"` is undefined, and it can lead to unexpected behaviour.
+
+### p2p.dial_timeout
+Timeout duration for the low-level dialer that connects to the remote address on the TCP network.
+```toml
+dial_timeout = "3s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+This parameter is the timeout value for dialing on TCP networks. If a hostname is used instead of an IP address and the
+hostname resolves to multiple IP addresses, the timeout is spread over each consecutive dial, such that each is given an
+appropriate fraction of the time to connect.
+
+Setting the value to `"0s"` disables timeout.
+
+## Mempool
+Mempool allows gathering and broadcasting uncommitted transactions among nodes.
+
+The **mempool** is a storage for uncommitted transactions; the **mempool cache** is an internal storage within the
+mempool, for invalid transactions. The mempool cache provides a list of known invalid transactions to filter out
+incoming duplicate transactions without running a full validation on them.
+
+### mempool.type
+The type of mempool this node will use.
+```toml
+type = "flood"
+```
+
+| Value type          | string    |
+|:--------------------|:----------|
+| **Possible values** | `"flood"` |
+|                     | `"nop"`   |
+
+`"flood"` is the original mempool implemented for CometBFT. It is a concurrent linked list with flooding gossip
+protocol.
+
+`"nop"` is a "no operation" or disabled mempool, where the ABCI application is responsible storing, disseminating and
```

Comment:
May a link to the issue introducing it?

Reply:
I think we should avoid having links to individual features implemented. If that's OK I'm not adding the link here.